include/llvm/InitializePasses.h

```diff
@@ -336,6 +336,69 @@ void initializeVirtRegMapPass(PassRegistry&);
 void initializeVirtRegRewriterPass(PassRegistry&);
 void initializeWholeProgramDevirtPass(PassRegistry &);
 void initializeWinEHPreparePass(PassRegistry&);
+
+// @LOCALMOD-BEGIN
+void initializeAddPNaClExternalDeclsPass(PassRegistry&);
```

Comment:
So just to confirm, we currently believe that these passes will *only* run for the emscripten target? That is, if we generate x86_64 linux code, these'll all be turned off by default?